README.md

```diff
@@ -876,6 +877,214 @@ See [here](https://www.typescriptlang.org) for further details.
 
 ## Advanced Features
 
+### Partitioned Databases
+
+Partitioned databases introduce the ability for a user to create logical groups
+of documents called partitions by providing a partition key with each document.
+
+Ensure your Cloudant cluster has the partitions feature enabled. A full list of
```

Comment:
Should we add that partitioned database is enabled when `{"partitioned":true}` is shown under the database's `props` metadata?